README.rst

```diff
@@ -43,16 +27,30 @@ expert or developer!
 
 Citing this software
 --------------------
-If you use this software for a publication, please cite the proper version using the following DOIs:
 
-- v0.14.0 : |doiv014|
-- v0.13.0 : |doiv013|
-- v0.12.0 : |doiv012|
-- v0.11.0 : |doiv011|
-- v0.10.5 : |doiv010|
-- v0.9.1 : |doiv09|
-- v0.8.0 : |doiv08|
-- v0.7.0 : |doiv07|
+If you use this software for a publication, please cite the Zenodo Record
+for the specific version you are using and our latest publication.
+
+You can find all ctapipe Zenodo records here: `List of ctapipe Records on Zenodo <https://zenodo.org/search?q=conceptrecid:%223372210%22&sort=-version&all_versions=True>`_.
+
+There is also a Zenodo DOI always pointing to the latest version: |doilatest|
+
+At this point, our latest publication is the 2021 ICRS proceeding, which you can
+cite using this bibtex entry:
+
+.. code::
+
+  @inproceedings{ctapipe-icrc-2021,
+      author = {Nöthe, Maximilian  and  Kosack, Karl  and  Nickel, Lukas  and  Peresano, Michele},
+      title = {Prototype Open Event Reconstruction Pipeline for the Cherenkov Telescope Array},
+      doi = {10.22323/1.395.0744},
+      booktitle = {Proceedings, 37th International Cosmic Ray Conference},
+      year=2021,
+      volume={395},
+      number={744},
+      location={Berlin, Germany},
+    }
+
 
 Installation for Users
```

Comment:
I think we don't really need this part in the README when we can just point to the actual documentation

Reply:
(I mean the installation instructions - from the preview it seems I refer to the citation)

Reply:
The documentation is more for developers, not for users (though probably that should be updated at some point to have both)

Reply:
Let's merge this and I'll make another PR updating the installation instructions